cmd/stand/main.go

```diff
@@ -38,11 +39,16 @@ func main() {
 
 //initCfg initialize configs
 func initCfg() (*config.Configs, error) {
```

Comment:
https://github.com/shinofara/stand/pull/76/files#diff-89ae71c9aea98e27909c4de6373e2e89R15 and https://github.com/shinofara/stand/pull/76/files#diff-89ae71c9aea98e27909c4de6373e2e89R16 variables.

It may be better to limit the scope.
For example,

```
var flCfgPath string
```